### DISCUSSION TITLE
How to pass props to the canvas?

### DISCUSSION THREAD

**Discussion by marcopiii** [bibixx/react-adobe-animate#68] [Q&A / Help] [answered]
As the documentation says, the props of `AnimationCC` are passed to the `div` surrounding the `canvas`. This includes the `style` prop.

Is there any way to pass a prop (for example a style) to the `canvas`?

**Comment by bibixx** [comment, marked as answer]
Hi @marcopiii 
Currently there is no way to pass any additional props to the `canvas` element.
To change it's styling (which isn't recommended as it may break some user interactions, eg. on click actions defined inside Adobe Animate editor) I'd suggest to create a css class and in it define the styling for canvas.

```css
.your-class-name > canvas {
  outline: 2px solid green;
}
```

```tsx
<AnimateCC
  animationName="lishtml5"
  composition="C1475B64B160904BB90B34246A5FF54B"
  paused={paused}
  className="your-class-name"
/>
```

**Comment by marcopiii** [comment]
@bibixx thanks for the clarification.
Let me point out that `AnimateCC` doesn't seem to have a `className` property (but this can be easily circumvented by targeting an element higher in the hierarchy).

**Reply by bibixx** [reply]
🤔 I must have done the TS typings incorrectly as looking at the code it should be passing the `className` property.
